### PR TITLE
Guide copy edits

### PR DESCRIPTION
**Pull request by kllevin**
#4 Updated site and existing guides with spelling and grammar edits.

### CODE REVIEW

File: source/git-install.html.erb [hunk 1]
```diff
@@ -11,34 +11,34 @@ title: Elixir Girls Installing Git
...
-            <h2>Configuring Git</h2>
+            <h2>Step 2 - Configure Git</h2>
             <% code("bash") do %>
 git config --global user.name "Your Actual Name"
 git config --global user.email "Your Actual Email"
             <% end %>
 
-            <h2>Github</h2>
-            <p>If you dont already have a Github account you will need to set one up if you want to deploy you application. It's free and only takes a couple of minutes please go to <a href="https://github.com" target="blank" class="inline-link">github.com</a>
+            <h2>Step 4 - Create a Github account</h2>
```
Comment (catherine-jones) [inline source/git-install.html.erb]: are we missing step 3?

**Review by catherine-jones — COMMENTED** [review]
I can't believe how many issues there were. I just made 1 comment I think I have misnumbered something that will need to be fixed. Thanks so much.

**Review by catherine-jones — APPROVED** [review]
Thanks

**Comment by kllevin** [comment]
Fixed - that was totally me not being able to count :)

**Pull request merged**